etl/steps/data/garden/hmd/2023-09-19/hmd.meta.yml

```diff
@@ -0,0 +1,84 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/dataset/
+dataset:
+  update_period_days: 365
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/en/latest/architecture/metadata/reference/tables/
+tables:
+  hmd:
+    variables:
+      central_death_rate:
+        title: Central death rate
+        description_short: Central death rate between ages x and x+n.
+        unit: deaths per people
+      probability_of_death:
+        title: Probability of death
```

Comment:
Suggest multiplying these values by 100 and having this as a % - currently the values are so small they don't display well on grapher.

Reply:
Great suggestion. I'll look into this in a future PR, might change what is displayed in the grapher dataset.